### ISSUE TITLE
复现指数太低

### ISSUE THREAD

**Issue by 620ztt** [likyoo/open-cd#144]
请问python tools/test.py configs/changer/changer_ex_r18_512x512_40k_levircd.py changer_r18_levir_workdir/latest.pth的latest.pth是指iter_40000.pth
，还是best_mIoU_iter_16000.pth，
使用latest.pth的指标很低，

![Image](https://github.com/user-attachments/assets/c57254b5-1ac0-41f5-85d0-259bc28cdbd1)

使用best_mIoU_iter_16000.pth的指标为

![Image](https://github.com/user-attachments/assets/1deb5190-cfd0-4044-8553-13cdbe55e0bf)这是什么原因

**Comment by Camille7911** [comment]
请问你解决了吗？我初步认为是标签数值问题，mmseg这个包的标签貌似都是(0,0,1)这种，但是咱们下载的label是(0,0,255),作者应该说明一下这个问题或者直接给出处理好的数据集文件，希望作者大大可以跟进一下这个问题。

**Comment by Camille7911** [comment]
Have you solved it? I initially think it’s a label value issue. In the mmseg package, the label mask values are like (0,0,1), but the labels we downloaded are (0,0,255). The author should explain this problem or directly provide properly processed dataset files. Hopefully, the author can pay attention to this issue.

**Comment by gl2yx** [comment]
> Have you solved it? I initially think it’s a label value issue. In the mmseg package, the label mask values are like (0,0,1), but the labels we downloaded are (0,0,255). The author should explain this problem or directly provide properly processed dataset files. Hopefully, the author can pay attention to this issue.

For datasets with labels in the range of [0, 255], it is necessary to add format_seg_map='to_binary' in the dataloader. For example:
train_dataloader = dict(
    batch_size=8,
    num_workers=4,
    persistent_workers=True,
    sampler=dict(type='InfiniteSampler', shuffle=True),
    dataset=dict(
        type='LEVIR_CD_Dataset',
       format_seg_map='to_binary',   # ←Here
        data_root=data_root,
        data_prefix=dict(
            seg_map_path='train/label',
            img_path_from='train/A', 
            img_path_to='train/B'),
        pipeline=train_pipeline))
